public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -173,6 +211,15 @@ define(function(require) {
       buttons.parent.removeClass("hide");
 
       publisher.updateDialog("");
+
+      Project.publishNeedsUpdate(false, function(err) {
+        if(err) {
+          console.error("[Bramble] Failed to set the publishNeedsUpdate flag after unpublishing with: ", err);
+          return;
```

Comment:
In other cases I don't have code running after the if statement so I don't need to return